htdocs/ecm/class/htmlecm.form.class.php

```diff
@@ -55,16 +55,20 @@ public function __construct($db)
 	 *  @param	int		$selected    		Id of preselected section
 	 *  @param  string	$select_name		Name of HTML select component
 	 *  @param	string	$module				Module ('ecm', 'medias', ...)
+	 *  @param	array	$ignore_id			Arroy of id to ignore
 	 *  @return	string						String with HTML select
 	 */
-	public function selectAllSections($selected = 0, $select_name = '', $module = 'ecm')
+	public function selectAllSections($selected = 0, $select_name = '', $module = 'ecm', $ignore_id = array())
```

Comment:
Can you rename the var 
$ignore_id
into
$ids_to_ignore

Because var ending with _id are used for a numeric ID so this create a confusion because here it is an array of several.

Reply:
It's done